src/lib/currency_codes.json

```diff
@@ -34,7 +34,7 @@
     "iso_code": "ALL",
     "name": "Albanian Lek",
     "symbol": "L",
-    "disambiguate_symbol": "Lek",
+    "disambiguate_symbol": "ALL ",
```

Comment:
Note I spoke with Kim about the new brand guide and she proposed to change some of the disambiguate_symbol.